plugins/nf-schema/src/main/nextflow/validation/config/SummaryConfig.groovy

```diff
@@ -1,21 +1,15 @@
-package nextflow.validation
+package nextflow.validation.config
 
 import groovy.util.logging.Slf4j
-import groovy.transform.PackageScope
-
 
 /**
  * This class allows to model a specific configuration, extracting values from a map and converting
  *
- * We anotate this class as @PackageScope to restrict the access of their methods only to class in the
- * same package
- *
  * @author : nvnieuwk <nicolas.vannieuwkerke@ugent.be>
  *
  */
 
 @Slf4j
-@PackageScope
```

Comment:
what is the reason to remove this?

Reply:
I moved the configs to a separate package called `nextflow.validation.config` so I couldn't access these classes anymore in the main package or other subpackages :)